### PR TITLE
Search input width is same as parent div.

### LINKED ISSUE #45: Search input should be the same width of the parent div
When focused the search bar is noticeably shorter than the parent container - these could be the same width just to look nice!

Screen Shot 2022-12-12 at 10 55 53 AM

### PR DESCRIPTION
**Pull request by Rishav1707**
### This PR fixes the following issues:-
closes #45 
### Changes made:-

- Replacing the padding property from 10px to 15px.
- Adding the margin-left property.
### Before
 
207106190-8ac9af98-7518-47fa-861a-e83dc10e1a76
### After
WhatsApp Image 2022-12-13 at 3 18 11 PM

### CODE REVIEW

**Review by amy-corson — CHANGES_REQUESTED** [review]
Hi Rishav! First of all, thanks so much for contributing to this project! We're really excited to have you onboard.

This solution is a little off the mark, so I just wanted to point out a few things that will help you fix this issue. 

1) You've targeted the right selector in CSS, but we're using SCSS to compile our CSS file, so you'll need to add these changes into the SCSS file and then use an extension such as Live Sass Compiler.
2) You're adding `padding` and a `margin-left` to the search bar, which is essentially adding pixels to all four sides and pushing it 2 pixels to the right. And while this may look okay on smaller screens, it's not showing up as the full width across the board. I would recommend targeting the input's `width` and try setting it according to the width of the parent container.
Screen Shot 2022-12-13 at 8 37 30 PM

**Comment by Rishav1707** [comment]
> Hi Rishav! First of all, thanks so much for contributing to this project! We're really excited to have you onboard.
> 
> This solution is a little off the mark, so I just wanted to point out a few things that will help you fix this issue.
> 
> 1. You've targeted the right selector in CSS, but we're using SCSS to compile our CSS file, so you'll need to add these changes into the SCSS file and then use an extension such as Live Sass Compiler.
> 2. You're adding `padding` and a `margin-left` to the search bar, which is essentially adding pixels to all four sides and pushing it 2 pixels to the right. And while this may look okay on smaller screens, it's not showing up as the full width across the board. I would recommend targeting the input's `width` and try setting it according to the width of the parent container.
> 
> Screen Shot 2022-12-13 at 8 37 30 PM

Done @amy-corson 
WhatsApp Image 2022-12-15 at 6 19 45 PM

**Comment by Rishav1707** [comment]
@amy-corson Please review my changes...

**Comment by Rishav1707** [comment]
> @amy-corson Please review my changes...

Anyone is there?? @amy-corson

**Comment by amy-corson** [comment]
> > @amy-corson Please review my changes...
> 
> Anyone is there?? @amy-corson

@Rishav1707  Hello! I do not work on this project full time. I see your pull request, I will review it when I get the chance. No need to keep tagging me. Thanks for your patience!

**Comment by Rishav1707** [comment]
> > > @amy-corson Please review my changes...
> > 
> > 
> > Anyone is there?? @amy-corson
> 
> @Rishav1707 Hello! I do not work on this project full time. I see your pull request, I will review it when I get the chance. Thanks for your patience!

OK

File: src/scss/_search.scss [hunk 1]
```diff
@@ -8,8 +8,9 @@
     border-radius: 10px;
   
     input {
-      padding: 10px;
+      padding: 15px;
```
Comment (amy-corson) [inline src/scss/_search.scss:11]: I would remove this, as the padding does not need to change for this fix.

File: src/scss/_search.scss [hunk 2]
```diff
@@ -8,8 +8,9 @@
...
       border: none;
+      width: 85%;
```
Comment (amy-corson) [inline src/scss/_search.scss:13]: This is really close! I would actually recommend a fix closer to: 
```
width: 100%;
box-sizing: border-box;
```
The width will make sure the input fills the entire parent container, and the box-sizing attribute will make sure it does not overflow.

Let me know if you have questions, and thanks for contributing

**Review by amy-corson — COMMENTED** [review]
This is closer! I recommended a few tweaks, but if you can add those changes you should be good to merge. Thanks so much!

**Review by amy-corson — COMMENTED** [review]
Hello! Thanks for your changes on this. The SCSS needs to be compiled into CSS for this change to take effect. You can use a compiler such as Watch Sass for this, or if you'd like I can add a commit to your branch. Let me know which one you'd prefer!

**Comment by Rishav1707** [comment]
> Hello! Thanks for your changes on this. The SCSS needs to be compiled into CSS for this change to take effect. You can use a compiler such as Watch Sass for this, or if you'd like I can add a commit to your branch. Let me know which one you'd prefer!

Sure, you can add a commit in this branch

**Review by amy-corson — APPROVED** [review]
Hello! this is good to merge. Apologies for the delay in getting back to you. You can ignore the substantial diff, my formatter went a little crazy

**Comment by Rishav1707** [comment]
When this PR get merged?

**Comment by amy-corson** [comment]
Thanks for the help!

**Pull request merged**